storage/libradb/src/lib.rs

```diff
@@ -93,6 +94,15 @@ fn error_if_too_many_requested(num_requested: u64, max_allowed: u64) -> Result<(
     }
 }
 
+fn gen_rocksdb_options(config: &RocksdbConfig) -> Options {
+    let mut db_opts = Options::default();
+    db_opts.create_if_missing(config.create_if_missing);
+    db_opts.create_missing_column_families(config.create_if_missing_column_families);
+    db_opts.set_max_open_files(config.max_open_files);
+    db_opts.set_max_total_wal_size(config.max_total_wal_size);
+    db_opts
+}
+
```

Comment:
can we use an std::convert::Into instead?

Reply:
I don't think so. Please tell me where it should be. I weighed different solutions. 
the `Into` trait must be implemented in either crate, rocksdb binding (which is third party) or libra-config (do you want to incorporate rocksdb dependency into `libra-config`?)